rectools/models/sasrec.py

```diff
@@ -600,15 +586,21 @@ def __init__(
             transformer_layers_type=transformer_layers_type,
             item_net_type=item_net_type,
         )
-        self.trainer = Trainer(  # TODO: move to lightning trainer and add option to pass initialized trainer
-            lr=lr,
-            epochs=epochs,
-            device=self.device,
-            loss=loss,
+        self.lightning_model: SasRec
+        self.trainer: Trainer
+        self._trainer = Trainer(
```

Comment:
Can we pass `logger = verbose > 0` here? To hide all cell outputs and to prevent lightning logging? Please check and add if it will work.